check_api/src/main/java/com/google/errorprone/util/ASTHelpers.java

```diff
@@ -798,6 +798,13 @@ private static boolean isFinal(Symbol symbol) {
     return (symbol.flags() & Flags.FINAL) == Flags.FINAL;
   }
 
+  private static final long RECORD_FLAG = 1L << 61;
```

Comment:
```suggestion
  /**
   * Flag for a record class, its canonical constructor and components. Can be replaced by
   * com.sun.tools.javac.code.Flags.RECORD once support minimum JDK version is 14.
   */
  private static final long RECORD_FLAG = 1L << 61;
```

Reply:
Although I really like the suggestion, I used this one-liner because it's used in the Error Prone project twice like that and wanted to be consistent.

So I'd lean towards leaving it as-is. (When we use this in `error-prone-support` we should definitely use this as Javadoc.

Reply:
Oh I misread, this is in the `ASTHelpers` of course... In that case, it _is_ a good idea to add this. Will commit that.